test/test.js

```diff
@@ -928,6 +928,16 @@ test( "bypassing validation on form submission", function() {
 	equal( $v.numberOfInvalids(), 1, "Validation failed correctly" );
 } );
 
+test( "works on contenteditable fields", function( assert ) {
+	var form = $( "#contenteditableForm" );
+	form.valid();
+	assert.hasError( $( "#contenteditableNumberInvalid" ), "Please enter a valid number." );
+	assert.hasError( $( "#contenteditableRequiredInvalid" ) , "This field is required." );
```

Comment:
The build is failed because of the illegal white space at this line:
```js
assert.hasError( $( "#contenteditableRequiredInvalid" ) , "This field is required." );
                                                       ⬆
```
Please remove it.